### ISSUE TITLE
run_net.py for webcam demo

### ISSUE THREAD

**Issue by alitirmizi23** [facebookresearch/SlowFast#569]
I am simply trying to run a webcam demo with I3D_8x8_R50.yaml as the configuration file. its as below:

`

TRAIN:
  ENABLE: False
  DATASET: kinetics
  BATCH_SIZE: 64
  EVAL_PERIOD: 10
  CHECKPOINT_PERIOD: 1
  AUTO_RESUME: True
DATA:
  NUM_FRAMES: 8
  SAMPLING_RATE: 8
  TRAIN_JITTER_SCALES: [256, 320]
  TRAIN_CROP_SIZE: 224
  TEST_CROP_SIZE: 256
  INPUT_CHANNEL_NUM: [3]
RESNET:
  ZERO_INIT_FINAL_BN: True
  WIDTH_PER_GROUP: 64
  NUM_GROUPS: 1
  DEPTH: 50
  TRANS_FUNC: bottleneck_transform
  STRIDE_1X1: False
  NUM_BLOCK_TEMP_KERNEL: [[3], [4], [6], [3]]
NONLOCAL:
  LOCATION: [[[]], [[]], [[]], [[]]]
  GROUP: [[1], [1], [1], [1]]
  INSTANTIATION: softmax
BN:
  USE_PRECISE_STATS: True
  NUM_BATCHES_PRECISE: 200
SOLVER:
  BASE_LR: 0.1
  LR_POLICY: cosine
  MAX_EPOCH: 196
  MOMENTUM: 0.9
  WEIGHT_DECAY: 1e-4
  WARMUP_EPOCHS: 34.0
  WARMUP_START_LR: 0.01
  OPTIMIZING_METHOD: sgd
MODEL:
  NUM_CLASSES: 400
  ARCH: i3d
  MODEL_NAME: ResNet
  LOSS_FUNC: cross_entropy
  DROPOUT_RATE: 0.5
TEST:
  ENABLE: False
  DATASET: kinetics
  BATCH_SIZE: 64
  CHECKPOINT_FILE_PATH: "checkpoints/I3D_8x8_R50.pyth"
DATA_LOADER:
  NUM_WORKERS: 8
  PIN_MEMORY: True
NUM_GPUS: 1
NUM_SHARDS: 1
RNG_SEED: 0
OUTPUT_DIR: .
DEMO:
  ENABLE: True
  LABEL_FILE_PATH: "kinetics400.json"
  WEBCAM: 0
  OUTPUT_FILE: "check.mp4"
  THREAD_ENABLE: True
  NUM_VIS_INSTANCES: 16
  NUM_CLIPS_SKIP: 1

`

But the **model weights don't get loaded** as shown below:

> 07/30 06:56:17][INFO] checkpoint.py:  220: Loading network weights from checkpoints/I3D_8x8_R50.pyth.
[07/30 06:56:17][INFO] checkpoint.py:  345: Network weights s1.pathway0_stem.conv.weight not loaded.
[07/30 06:56:17][INFO] checkpoint.py:  345: Network weights s1.pathway0_stem.bn.weight not loaded.
[07/30 06:56:17][INFO] checkpoint.py:  345: Network weights s1.pathway0_stem.bn.bias not loaded.
[07/30 06:56:17][INFO] checkpoint.py:  345: Network weights s1.pathway0_stem.bn.running_mean not loaded.
[07/30 06:56:17][INFO] checkpoint.py:  345: Network weights s1.pathway0_stem.bn.running_var not loaded.
[07/30 06:56:17][INFO] checkpoint.py:  345: Network weights s1.pathway0_stem.bn.num_batches_tracked not loaded.
[07/30 06:56:17][INFO] checkpoint.py:  345: Network weights s2.pathway0_res0.branch1.weight not loaded.
[07/30 06:56:17][INFO] checkpoint.py:  345: Network weights s2.pathway0_res0.branch1_bn.weight not loaded.
[07/30 06:56:17][INFO] checkpoint.py:  345: Network weights s2.pathway0_res0.branch1_bn.bias not loaded.
[07/30 06:56:17][INFO] checkpoint.py:  345: Network weights s2.pathway0_res0.branch1_bn.running_mean not loaded.
[07/30 06:56:17][INFO] checkpoint.py:  345: Network weights s2.pathway0_res0.branch1_bn.running_var not loaded.
[07/30 06:56:17][INFO] checkpoint.py:  345: Network weights s2.pathway0_res0.branch1_bn.num_batches_tracked not loaded.
[07/30 06:56:17][INFO] checkpoint.py:  345: Network weights s2.pathway0_res0.branch2.a.weight not loaded.
[07/30 06:56:17][INFO] checkpoint.py:  345: Network weights s2.pathway0_res0.branch2.a_bn.weight not loaded.
[07/30 06:56:17][INFO] checkpoint.py:  345: Network weights s2.pathway0_res0.branch2.a_bn.bias not loaded.
[07/30 06:56:17][INFO] checkpoint.py:  345: Network weights s2.pathway0_res0.branch2.a_bn.running_mean not loaded.
[07/30 06:56:17][INFO] checkpoint.py:  345: Network weights s2.pathway0_res0.branch2.a_bn.running_var not loaded.
[07/30 06:56:17][INFO] checkpoint.py:  345: Network weights s2.pathway0_res0.branch2.a_bn.num_batches_tracked not loaded.
[07/30 06:56:17][INFO] checkpoint.py:  345: Network weights s2.pathway0_res0.branch2.b.weight not loaded.
[07/30 06:56:17][INFO] checkpoint.py:  345: Network weights s2.pathway0_res0.branch2.b_bn.weight not loaded.
[07/30 06:56:17][INFO] checkpoint.py:  345: Network weights s2.pathway0_res0.branch2.b_bn.bias not loaded.
[07/30 06:56:17][INFO] checkpoint.py:  345: Network weights s2.pathway0_res0.branch2.b_bn.running_mean not loaded.
[07/30 06:56:17][INFO] checkpoint.py:  345: Network weights s2.pathway0_res0.branch2.b_bn.running_var not loaded.
[07/30 06:56:17][INFO] checkpoint.py:  345: Network weights s2.pathway0_res0.branch2.b_bn.num_batches_tracked not loaded.
[07/30 06:56:17][INFO] checkpoint.py:  345: Network weights s2.pathway0_res0.branch2.c.weight not loaded.
[07/30 06:56:17][INFO] checkpoint.py:  345: Network weights s2.pathway0_res0.branch2.c_bn.weight not loaded.

Any idea why this would be happening?

**Comment by alitirmizi23** [comment]
Maybe there's something in the defaults.py that I might be missing; it just won't load the weights. 
Any help?
